File: back-end/twitter.rb
# connect to Twitter API
require "Twitter"
client = Twitter::REST::Client.new do |config|
    config.consumer_key = "5sOOFfHHPYjiPQyeBcTrPeZhl"
    config.consumer_secret = "nf8d6tIDEPwiQI8iBfMA0272KrK4LeEKrHMNagMuMQOjrg9LRE"
    config.access_token = "998982729021603840-hn5G0obTyAwtdl2zUXVn2TvtFPFeTnQ"
    config.access_token_secret = "GU0HuihhLHSRtqoKYtOTjE5kUKJdYGSsd73jeOjqy3K5a"
end
client.update('Hello World!')

File: src/api/twitter_func/twitter.js
// twitter dev account keys to access API
require('dotenv').config()
const twit = require('twit')
const twitterAPI = new twit({
    consumer_key : process.env.api_key,
    consumer_secret : process.env.api_secret,
    access_token : process.env.access_token,
    access_token_secret : process.env.access_token_secret
});
module.exports = {twitterAPI}